rskj-core/src/test/java/co/rsk/peg/BridgeSupportTest.java

```diff
@@ -7577,6 +7577,35 @@ public void getNextPegoutCreationBlockNumber_after_RSKIP271_activation() {
         assertEquals(10L, bridgeSupport.getNextPegoutCreationBlockNumber());
     }
 
+    @Test
+    public void getQueuedPegoutsCount_before_RSKIP271_activation() {
+
+        ActivationConfig.ForBlock activations = mock(ActivationConfig.ForBlock.class);
+        when(activations.isActive(ConsensusRule.RSKIP271)).thenReturn(false);
+
+        BridgeSupport bridgeSupport = bridgeSupportBuilder
+                .withActivations(activations)
+                .build();
+
+        assertEquals(0, bridgeSupport.getQueuedPegoutsCount());
+    }
```

Comment:
Let's add a mock for BridgeStorageProvider when creating BridgeSupport. Then verify `provider..getReleaseRequestQueueSize()` was never called.

Just to be sure

Reply:
Resolved

Reply:
I meant to add this in the pre RSKIP activation test.

`BridgeSupport bridgeSupport = bridgeSupportBuilder.withActivations(activations).withProvider(...)`

And then `verify(times(0), provider..getReleaseRequestQueueSize())`